google/cloud/aiplatform/utils.py

```diff
@@ -471,6 +472,14 @@ class MetadataClientWithOverride(ClientWithOverride):
     )
 
 
+class TensorboardClientWithOverride(ClientWithOverride):
+    _is_temporary = True
```

Comment:
```suggestion
    _is_temporary = False
```

This will keep the grpc connection alive between calls.

Reply:
Awesome applied.